set.mm

```diff
@@ -679284,7 +679293,7 @@ base set if and only if the neighborhoods (convergents) of every point
                     u - v ) ) ) = ( 2 x. ( ( F ` u ) x. ( G ` v ) ) ) ) $.
     imo72b2.6 $e |- ( ph -> A. y e. RR ( abs ` ( F ` y ) ) <_ 1 ) $.
     imo72b2.7 $e |- ( ph -> E. x e. RR ( F ` x ) =/= 0 ) $.
-    $( IMO 1972 B2.  (14th International Mathemahics Olympiad in Poland,
+    $( IMO 1972 B2.  (14th International Mathemathics Olympiad in Poland,
```

Comment:
```suggestion
    $( IMO 1972 B2.  (14th International Mathematical Olympiad in Poland,
```
searching it up online, this is the actual acronym for the IMO, whose 14th occurrence did indeed happen in Poland: https://en.wikipedia.org/wiki/List_of_International_Mathematical_Olympiads